packages/video_player/video_player_web/example/README.md

```diff
@@ -0,0 +1,21 @@
+# Testing
+
+This package utilizes the `integration_test` package to run its tests in a web browser.
+
+See [flutter.dev > Integration testing](https://flutter.dev/docs/testing/integration-tests) for more info.
+
+## Running the tests
```

Comment:
We should add this to https://github.com/flutter/flutter/wiki/Plugin-Tests#integration-tests (which I created recently as part of overhauling the contributing guide) so that we can just refer to it from READMEs instead of having to duplicate it everywhere.

Reply:
Closing the loop here: added https://github.com/flutter/flutter/wiki/Plugin-Tests#web-tests